src/modules/job-info/guest_watch.c

```diff
@@ -43,7 +43,7 @@
  *
  * 2) If the guest namespace is already copied into the main namespace
  *    (event "release" and "final=true"), we watch the eventlog in the
- *    main namespace (main_namespace_lookup()).  This is the "easy" case
+ *    main namespace (main_namespace_watch()).  This is the "easy" case
```

Comment:
Wondering about this name change - once the guest info is copied to the primary namespace, it is static, but "watch" implies that it is changing.

Reply:
yeah, i did go back and forth on it.  I went with `watch` b/c I go through `job-info.eventlog-watch` instead of calling `kvs.lookup` (similarly `guest_namespace_watch` also goes through `job-info.eventlog-watch`).   

I did consider going with `stream` instead, but that sort of describes internal knowledge of what `job-info.eventlog-watch` is doing.